lib/vector-buffers/src/variants/disk_v2/writer.rs

```diff
@@ -1352,7 +1352,7 @@ where
 {
     /// Closes this [`Writer`], marking it as done.
     ///
-    /// Closing the writer signals to the reader that that no more records will be written until the
+    /// Closing the writer signals to the reader that no more records will be written until the
```

Comment:
often `that that` is intentional and valuable -- here it doesn't seem to be and it's problematic